### ISSUE TITLE
Add Strava to HA will not work

### ISSUE THREAD

**Issue by JePeSWE** [codingcyclist/ha_strava#47]
Hi, when i´m trying to add the github https://github.com/codingcyclist/ha_strava i ge this error msg:

<Integration codingcyclist/ha_strava> Repository structure for 0.1.1 is not compliant

What am i doing wrong.?

**Comment by ombarres** [comment]
the same...

**Comment by lennon101** [comment]
Yeah same for me. I don't think we are doing anything wrong. It is as the error reports: "the folder structure is not compliant". So the OP will need to update it. Fingers crossed he sees this thread 🤞
@codingcyclist

**Comment by lennon101** [comment]
@codingcyclist

**Comment by thoemmi** [comment]
This repo is abandoned. I recommend you try @craibo's fork at https://github.com/craibo/ha_strava which is actively maintained.

**Comment by jjmateof** [comment]
Hello,

Any idea how uninstall Strava HA from HACS? I tried to delete the app from hacs directory and did not work.

Regards.

**Comment by craibo** [comment]
Hi @jjmateof 

1. From _Settings -> Devices & Integrations_ find the Strava integration. 
1.1. Select the 3 dots menu bottom right and select the _Delete_ menu item
2. Navigate to _HACS -> Integrations_ view and select the HA Strava tile.
2.1. In the HA Strava view, select the 3 dots menu top right and select _Remove_ menu item
3. Go back to _HACS -> Integrations_ view
3.1. Select the 3 dots menu and select _Custom repositories_ menu item
3.2. Find the Strava repository and delete it.
4. Finally go to _Settings -> System_ and select  _Restart_ top right

As mentioned previously, the repo https://github.com/craibo/ha_strava is a currently maintained and working version of HA Strava
